### DISCUSSION TITLE
Feature to colorize notification cards (based on app average color) on Hyper OS

### DISCUSSION THREAD

**Discussion by nhanntt0401** [MonwF/customiuizer#545] [Q&A]
As I remembered, this tweak used to have an option to colorize the notification cards based on app color. As I have an intention to go back to xiaomi, could anyone share me if this option still work on the mordern hyperos?

**Comment by MonwF** [comment]
it still works on hyperos 1

**Reply by nhanntt0401** [reply]
thank you so much

**Comment by nhanntt0401** [comment]
I have another question, does customiuizer work with kernelsu?
